connection.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+// Connection to the Database
+$servername = "localhost";
```

Comment:
let's move these in a `config` file that would be ignored in `.gitignore` and use these important variables as DEFINES.

Otherwise, If I set the project, it won't work on my machine, due to different username/pass